components/decentralized_dns/content/decentralized_dns_opt_in_page.cc

```diff
@@ -63,7 +63,7 @@ void DecentralizedDnsOptInPage::CommandReceived(const std::string& command) {
       controller()->Proceed();
       break;
     default:
-      NOTREACHED_IN_MIGRATION() << "Unsupported command: " << command;
+      NOTREACHED() << "Unsupported command: " << command;
   }
```

Comment:
Ditto

Reply:
Force pushed an update to fix this, https://github.com/brave/brave-core/commit/babdb91402f60ba31f30a258259bef6919149be9